.travis.yml

```diff
@@ -18,6 +21,15 @@ before_install:
 
 install:
   - dep ensure
+  - docker build -t eduboard/backend .
+  - docker run -d -p 127.0.0.1:8081:8080 --name eduboard eduboard/backend
 
 script:
- - go test -v -race -cover ./...
+  - docker ps | grep -q eduboard
+  - go test -v -race -cover ./...
+
+deploy:
```

Comment:
We actually don't need this. Dockerhub detects changes on master branch automatically and builds the image already.

Reply:
https://hub.docker.com/r/eduboard/backend/builds/

Reply:
Thats nice to know :D

But do we really want to publish an image to docker for every branch that we have?

Reply:
We don't. Only master.